engine-nextgen/src/main/kotlin/graphql/nadel/validation/NadelHydrationValidation.kt

```diff
@@ -233,4 +247,13 @@ internal class NadelHydrationValidation(
             }
         }
     }
+
+    /*
+    *  Checks the type of a hydration argument against the type of an actor field argument to see if they match
+    *
+    */
+    private fun hydrationArgTypesMatch(outputType: GraphQLOutputType, inputType: GraphQLInputType): Boolean {
+        //TODO
+        return true
```

Comment:
I think you should use `NadelTypeValidation.isAssignableTo` with lhs as `inputType` and rhs as `outputType`